pkg/ibmvpc/examples/out/analysis_out/acl_testing3_all_vpcs_.md

```diff
@@ -6,10 +6,10 @@
 | db-endpoint-gateway-ky[10.240.30.7] | vsi3a-ky[10.240.30.5] | All Connections |
 | db-endpoint-gateway-ky[10.240.30.7] | vsi3b-ky[10.240.30.6] | All Connections |
 | db-endpoint-gateway-ky[10.240.30.7] | vsi3c-ky[10.240.30.4] | All Connections |
-| vsi1-ky[10.240.10.4] | Public Internet 161.26.0.0/16 | protocol: UDP * |
+| vsi1-ky[10.240.10.4] | Public Internet 161.26.0.0/16 | protocol: UDP |
 | vsi1-ky[10.240.10.4] | vsi2-ky[10.240.20.4] | protocol: TCP,UDP |
 | vsi2-ky[10.240.20.4] | Public Internet (all ranges) | All Connections |
-| vsi2-ky[10.240.20.4] | vsi1-ky[10.240.10.4] | All Connections * |
+| vsi2-ky[10.240.20.4] | vsi1-ky[10.240.10.4] | All Connections |
```

Comment:
Before diving into the code - why was the "*" in this "All connections" (and in other similar places) line removed?
This connection, as well as the other "All connections" ones for which the "*" disappeared,  were marked with * only due to UDP, that is by mistake, in the first place?

Reply:
The `*` is now added only for TCP unidirectional connections. 
For other protocols this analysis does not apply. 
So in this example, the subset connection for which unidirectional was identified, was not within TCP. 
Thus, in the new analysis, the `*` is not added.

Reply:
So the statefullness information is given for a connection which is only TCP but is not given if the connection is both TCP and UDP? I'm not sure I'm following. Perhaps we should take this offline.

Reply:
It is given for the subset-contained-within-TCP  of the connection.